src/configuration.ts

```diff
@@ -584,3 +585,11 @@ export function convertRuleOptions(ruleConfiguration: Map<string, Partial<IOptio
     });
     return output;
 }
+
+export function isFileExcluded(filepath: string, configFile: IConfigurationFile) {
```

Comment:
make `configFile?` arg optional to support `undefined`. you already have a check for its existence.